src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,8 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
```

Comment:
'3' could be considered as a magic number here. So, I'd recommend creating a constant for this number as described in common mistakes